packages/embark/src/lib/modules/ethereum-blockchain-client/index.js

```diff
@@ -44,15 +47,16 @@ class EthereumBlockchainClient {
     }
 
     // this.blockchain.deployContractFromObject(deployObject,
-    console.dir({ arguments: contract.args, data: ("0x" + contract.code) });
+      console.dir({ arguments: contract.args, data: ("0x" + contract.code) });
```

Comment:
Same thing here. Maybe it's a Github issue

Reply:
Fixed, thanks!